### PR TITLE
Updates Outdated Entry Text, Requires CMO to play researcher

### PR DESCRIPTION
**Pull request by Crimsonerva**
# About the pull request
Updates entry text for all roles to be more accurate with modern standards.

Requires CMO to play 3 hours of doctor and 3 of researcher, and 1 additional medical role elsewhere.

# Explain why it's good for the game

A lot of the entry text has some outdated information, and this modernizes it to be in accordance with SOP/ML now, along with the rules. People are also told what place they are in for the chain of command, if they are set to take acting commander, outside of squad roles.

# Changelog
:cl: Ahana
balance: requires CMO to play researcher and doctor for 3 hours each now
balance: changes entry text in most roles to reflect more modern standards and duties
fix: fixed wrongly listed supervisors for certain marine roles
spellcheck: fixed sign referencing the requisitions officer in the requisitions line to now refer to the quartermaster
spellcheck: marine roles now refer to the leader of the operation as simplified 'commander' as said in marine law, rather than commanding officer to enhance clarity.

/:cl:

### CODE REVIEW

**Comment by Superreallycoolguy** [comment]
15 hours as ASO for XO....................you goofy goober, who let you cook?

 to try XO, you're gonna need uh 70 hours? I calculate that total right? I guess maybe 60 if you only play SL. I think that math is right. Wait no for ASO you need a total of 60 hours as well? Well its 100+ hours

  get outta here. Go, shoo, git. Nobody is gonna wait that long to play XO, you are beyond out of your mind. I don't care if we get a shitty XO now and then, no right human being is gonna take the time on this 2D game to get XO.

**Comment by hislittlecuzingames** [comment]
If SO could be hours on FTL OR SL, I'd say that's more reasonable.

**Comment by MobiusWon** [comment]
I think the scope went a bit too far here. In my opinion only the XO really needs a bump in playtime requirements, all other roles should be the same aa they are now (except maybe ASO?)

**Comment by Nanu308** [comment]
Unlocked by request from the PR author, behave in comment section or risk git sanctions

**Comment by Superwelder** [comment]
> ## Medbay
> 
> **CMO** must play researcher and doctor before playing CMO.
> 
> **Researchers** must play doctor/surgeon before playing researcher.

> 
> It encourages players to learn each role of their department before deciding to lead it. Currently, for instance, someone can play 15 hours of medic, and unlock CMO. This skips any surgical knowledge or research knowledge.
> 
> Chief Medical Officer
> 15 -> 21 hours total hours medical personnel
> Researcher
> 5 hours medic roles -> 3 hours doctor

At time of writing, #5685 is still awaiting approval or denial from staff. Should that PR be merged, the requirement for CMO to have played research may already be somewhat obsolete from an in character perspective. As CMO, you should no doubt have a grasp on what a research chem is capable of but that can be learned by playing medic or doctor for a couple rounds anyways. This does bring up the question of if CMO should still be in charge of publishing research chems but that is a topic for that particular PR. Regardless, changes to CMO should be held back until that PR has been decided upon.

**Review by Drulikar — CHANGES_REQUESTED** [review]
None of the proposed timelock changes are necessary. However if you want to revert those we can work through the other proposed changes to entry messages, descriptions, etc.

**Comment by Crimsonerva** [comment]
Timelocks were reverted, aside from the CMO one I cleared with you in Discord.

File: code/game/jobs/job/civilians/support/synthetic.dm
Comment (Drulikar) [inline code/game/jobs/job/civilians/support/synthetic.dm:12]: I'm not sure this is exactly correct or it seems awkward this way. I'd just leave it as the acting commander since that's going to be the value for most roles anyways.

Comment (Crimsonerva) [reply code/game/jobs/job/civilians/support/synthetic.dm:12]: Synth guidelines require players to follow command staff's orders.
"Do not disobey valid orders from Command personnel. An unlawful order is not valid, thus can be ignored."

I'll leave the final call up to you, I only had it this way because that specifically requires synthetics to follow it. I thought it would be a good reminder for newer synthetics.

File: code/game/jobs/job/command/auxiliary/auxiliary_support_officer.dm
Comment (Crimsonerva) [reply code/game/jobs/job/command/auxiliary/auxiliary_support_officer.dm:9]: I did. Some things got jumbled up it seems when I was reverting the other stuff. Fixed! 😄

**Comment by Crimsonerva** [comment]
All outstanding grammatical issues were revised and fixed. Let me know if there's anything else!

File: code/game/jobs/job/civilians/support/synthetic.dm
Comment (Drulikar) [reply code/game/jobs/job/civilians/support/synthetic.dm:12]: When I asked the council they said it was that you need to listen to 2nd Lt and above not that you answer to them directly. There were also concerns that people could abuse this wording. So no I still think it should be how it was.

File: code/game/jobs/job/command/cic/staffofficer.dm
Comment (Drulikar) [reply code/game/jobs/job/command/cic/staffofficer.dm:10]: This is still unresolved.

File: code/game/jobs/job/command/auxiliary/auxiliary_support_officer.dm
Comment (Drulikar) [reply code/game/jobs/job/command/auxiliary/auxiliary_support_officer.dm:9]: This is still unresolved.

**Review by Drulikar — CHANGES_REQUESTED** [review]
See the 3 outstanding issues.

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Pull request closed without merging**
